python/lsst/ts/envsensors/command_handler.py

```diff
@@ -266,8 +284,13 @@ async def connect_devices(self) -> None:
         device_configurations = self._configuration[Key.DEVICES]  # type: ignore
         self._devices = []
         for device_configuration in device_configurations:
-            device = self._get_device(device_configuration)
+            device: BaseDevice = self._get_device(device_configuration)
             self._devices.append(device)
+            self.log.info(
```

Comment:
This seems a rather inefficient way to log the configuration. Please consider logging a single message with the full list of sensors for which you are given configuration.

Or/and make it debug-level so it's not normally stored in the EFD.

Reply:
I made it debug-level since it is useful to see which sensors do and which don't get configured and/or started correctly.